content/en/monitors/monitor_types/forecasts.md

```diff
@@ -9,39 +9,42 @@ further_reading:
   text: "Configure your monitor notifications"
 - link: "monitors/downtimes"
   tag: "Documentation"
-  text: "Schedule a downtime to mute a monitor"
+  text: "Schedule downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
   text: "Consult your monitor status"
 ---
 
-Forecasting is an algorithmic feature that allows you to predict where
-a metric is heading in the future. It is well-suited for metrics with
-strong trends or recurring patterns.
+Forecasting is an algorithmic feature that allows you to predict where a metric is heading in the future. It is well-suited for metrics with strong trends or recurring patterns.
 
 For example, if your application starts logging at a faster rate, forecasts can alert you a week before your disk fills up, giving you adequate time to update your log rotation policy. Or, you can forecast business metrics, such as user sign-ups, to track progress against your quarterly targets.
 
+To use forecast monitoring, start by creating a [forecast timeseries graph](#forecast-dashboard), and then [set up a monitor](#forecast-monitoring) to trigger when a metric is forecasted to hit a specific threshold.
+
 ## Forecast function
 
 There is a `forecast` function in the Datadog query language. When you apply this function to a series, it returns the usual results along with a forecast of future values.
 
-## Forecast monitor
+## Forecast dashboard
 
-The chart below shows a dashboard tile of a forecast in "live" mode. The dashed pink line divides the history from the forecast, and the current time is indicated by the gray "Now" marker and line. The dotted green line shows the future values predicted by the forecast, and the range of predicted values within 3 deviations (a tunable parameter) is shown in lighter green.
+To create a forecast graph, start by adding a timeseries graph to your dashboard. Be sure to select **Timeseries** as the visualization type. Now, click on the **+** icon on the right side of your expression. Choose the **Forecast** function in the **Algorithms** submenu. You should immediately see the preview update include the forecast visualization. A number of the graphing options disappear, as forecasts have a unique visualization.
```

Comment:
```suggestion
To create a forecast graph, start by adding a timeseries graph to a dashboard or a timeseries cell to a notebook. Now, click on the **+** icon on the right side of your expression. Choose the **Forecast** function in the **Algorithms** submenu. You should immediately see the preview update to include the forecast visualization. A number of the graphing options disappear, as forecasts have a unique visualization.
```

Since the "Timeseries" is the default visualization type in dashboards, I think I'd rather sacrifice that clarification for the sake of generalizing this to also include notebooks.